constraint/pkg/client/drivers/k8scel/transform/make_vap_objects.go

```diff
@@ -79,14 +79,42 @@ func ConstraintToBinding(constraint *unstructured.Unstructured) (*admissionregis
 		return nil, err
 	}
 
-	var enforcementAction admissionregistrationv1beta1.ValidationAction
-	switch enforcementActionStr {
-	case apiconstraints.EnforcementActionDeny:
-		enforcementAction = admissionregistrationv1beta1.Deny
-	case "warn":
-		enforcementAction = admissionregistrationv1beta1.Warn
-	default:
-		return nil, fmt.Errorf("%w: unrecognized enforcement action %s, must be `warn` or `deny`", ErrBadEnforcementAction, enforcementActionStr)
+	actions := []string{}
+	if apiconstraints.IsEnforcementActionScoped(enforcementActionStr) {
+		actionsForEP, err := apiconstraints.GetEnforcementActionsForEP(constraint, []string{apiconstraints.WebhookEnforcementPoint})
+		if err != nil {
+			return nil, err
+		}
+		if len(actionsForEP[apiconstraints.WebhookEnforcementPoint]) == 0 {
```

Comment:
Should VAP be its own enforcement point?

Reply:
that makes sense and the actions will be mapped as: deny, warn, audit as part of the generated vap resource.

Reply:
does `validation.gatekeeper.sh -> gatekeeper webhook & admission.k8s.io -> VAP` work?

Reply:
Maybe? Probably best to bikeshed on naming at the community meeting.  Admission webhooks are their own concept in K8s, IIRC.

vap.k8s.io?